runatlantis.io/docs/server-configuration.md

```diff
@@ -450,7 +450,9 @@ Values are chosen in this order:
   # or
   ATLANTIS_GH_TEAM_ALLOWLIST="myteam:plan, secteam:apply, DevOps Team:apply"
   ```
-  Comma-separated list of GitHub team name (not a slug) and permission pairs. By default, any team can plan and apply.
+  In versions v0.21.0 and below, the GitHub team name required the case sensitive team name. Future versions will allow name or slug.
```

Comment:
Unsure if this will be merged prior to v0.21.0 release or after. I assumed after.

Reply:
I spoke with @jamengual and it was decided to merge this change in as a bugfix for the upcoming v0.21.0 release.